smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/RuleSetParameterFinder.java

```diff
@@ -152,6 +153,54 @@ public Map<String, String> getContextParams(Shape operationInput) {
         return map;
     }
 
+    /**
+     * Get map of params to JavaScript equivalent of provided JMESPath expressions.
+     */
+    public Map<String, String> getOperationContextParamValues(Shape operationInput) {
```

Comment:
please add unit tests for this

Reply:
Tests added:
* [`f022101` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/f0221019a66e7931efa9df77368adcfa6d1fc5df)
* [`b969772` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/b969772fa2697d4866b75478a20baccc29e253b5)
* [`1782bdb` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/1782bdbb03d995fd871a478bb9017c496cd34c9c)
* [`459774a` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/459774a9b846ac1901f4aa04ee26ff42d88760b5)